graphrag/index/operations/summarize_communities/strategies.py

```diff
@@ -65,31 +64,25 @@ async def _run_extractor(
         await rate_limiter.acquire()
         results = await extractor({"input_text": input})
         report = results.structured_output
-        if report is None or len(report.keys()) == 0:
+        if report is None:
             log.warning("No report found for community: %s", community)
             return None
 
         return CommunityReport(
             community=community,
             full_content=results.output,
             level=level,
-            rank=_parse_rank(report),
-            title=report.get("title", f"Community Report: {community}"),
-            rank_explanation=report.get("rating_explanation", ""),
-            summary=report.get("summary", ""),
-            findings=report.get("findings", []),
-            full_content_json=json.dumps(report, indent=4, ensure_ascii=False),
+            rank=report.rating,
+            title=report.title,
+            rank_explanation=report.rating_explanation,
+            summary=report.summary,
+            findings=[
+                Finding(explanation=f.explanation, summary=f.summary)
+                for f in report.findings
+            ],
+            full_content_json=report.model_dump_json(indent=4),
```

Comment:
I'm just curious, is this output supposed to be human-readable? I ask because of the indent; this might make us use more memory and disk space if we have a ton of community reports (I know this is probably not part of this PR)

Reply:
I think we were doing this for debug purposes months ago. We probably don't need human-readable output, but I'd probably move to defer it.